### PR TITLE
chore: replaces travis

### PR DESCRIPTION
**Pull request by paschalidi**
👋 Hey both. 

This replaces travis with the verify gh action. 
The reason is that travis charges public repos as it seems. There is a slack conversation about it [here](https://dhis2.slack.com/archives/CDX5CE15W/p1606319576064200).

Let me know your thoughts on this

### CODE REVIEW

File: src/core_modules/capture-core/metaDataMemoryStoreBuilders/programs/programsBuilder.js [hunk 1]
```diff
@@ -176,10 +176,16 @@ function getBuiltPrograms(
...
+        .filter((cachedProgram) => {
+            // allow event programs
+            if (cachedProgram.programType === 'WITHOUT_REGISTRATION') {
+                return true;
+            }
+            // We allow only tracker programs that their tracker id exists in the collection.
+            // This is because a user might have access to read and write to a program BUT
+            // they might not have access to read and write to the tracked entity type the program belongs to.
+            return (cachedProgram.trackedEntityTypeId && trackedEntityTypeCollection.get(cachedProgram.trackedEntityTypeId));
+        })
```
Comment (varl) [inline src/core_modules/capture-core/metaDataMemoryStoreBuilders/programs/programsBuilder.js:188]: Why is this change bundled with replacing travis?

Comment (paschalidi) [reply src/core_modules/capture-core/metaDataMemoryStoreBuilders/programs/programsBuilder.js:188]: whoops that shouldnt be here. was doing some branch juggling on friday and must have missed this bit

**Comment by paschalidi** [comment]
These are failing because I am rebuilding the `ca-test-2.36` https://awx.dhis2.org/#/jobs/playbook/19529?job_search=page_size:20;order_by:-finished;not__launch_type:sync

I will rerun the jobs once this job is complete

File: .github/workflows/verify-app.yml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+          node-version: 12.x
+
+      - uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Lint
+        run: yarn lint
```
Comment (JoakimSM) [inline .github/workflows/verify-app.yml:52]: shouldn't this be `yarn linter:check`?

**Review by varl — APPROVED** [review]

File: .github/workflows/verify-app.yml (hunk 2 above)
Comment (paschalidi) [reply .github/workflows/verify-app.yml:52]: yes it should

**Review by JoakimSM — APPROVED** [review]

**Pull request merged**